tooling/templatize/cmd/pipeline/validate/cmd.go

```diff
@@ -28,7 +31,17 @@ func NewCommand() (*cobra.Command, error) {
 		SilenceErrors: true,
 		SilenceUsage:  true,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return runInspect(cmd.Context(), opts)
+			if err := runValidate(cmd.Context(), opts); err != nil {
+				lines := strings.Split(err.Error(), "\n")
```

Comment:
The 300-line-long error message from the JSONSchema parsing is really not something you want to see in one line, with escaped `\n` characters ...